grader.js

```diff
@@ -506,12 +506,62 @@ var Grader = (function() {
       }
       var hasCorrectStyle = false;
       var currentStyle = elem.css(cssProperty);
-      if (currentStyle  === _correctStyle) {
+      if (currentStyle === _correctStyle) {
         hasCorrectStyle = true;
       }
       return hasCorrectStyle;
     },
 
+    hasStyleOtherThan : function (elem, cssProperty, incorrectStyle) {
```

Comment:
couldn't someone just use `!grader.hasCorrectStyle()` instead of this? not sure I see the need for this method